### ISSUE TITLE
expression-annotations/annotations/gtex/ directory cleaning

### ISSUE THREAD

**Issue by aechchiki** [BgeeDB/expression-annotations#124]
Hi, I think time has come to clean the expression-annotations/annotations/gtex/ directory, as @fbastian suggested some weeks ago. 

My guess is that we should delete all intermediary working files (aka, the ones referenced in #110), of course without deleting necessary files.

So, in my opinion, we should keep:
- `expression-annotations/annotations/gtex/GTEx_subjects-track.tsv` , which reports, for every original GTEx subject, information for all the variables we were interested in (see google docs) directly from the` subject_phenotypes` original file
- ` expression-annotations/annotations/gtex/GTEx_subject_annots.tsv`, which reports, for every original GTEx subject, a definition of inclusion status (to be annotated in Bgee: Y/N) and a simplified version of criteria for inclusion/exclusion from bgee (based on selective information reported in `./GTEx_subjects-track.tsv`)
- `expression-annotations/annotations/gtex/GTEx_samples-track.tsv`, which reports, for every original GTEx sample, a  a definition of inclusion status (to be annotated in Bgee: Y/N) and a summary of criteria for inclusion/exclusion from bgee (based on information reported in `./GTEx_subject_annots.tsv` and the `samples_attributes` original file 
- `expression-annotations/annotations/gtex/review_of_GTEx_annotations_FILES/selected-samples_updated-sorted.tsv`, which reports all samples tagged "yes" and sample-type "RNAseq-TrueSeq.v1", and was used for sample annotation in the final` RNASeqLibrary.tsv`
- `expression-annotations/annotations/gtex/review_of_GTEx_annotations_FILES/Uberon-mapping_GTEx-vs-Bgee.tsv`, which reports all the changes in Uberon mapping relatively to the original mapping by GTEx

Should we create first a subdirectory (eg. `expression-annotations/annotations/gtex/gtex_final`) in which to transfer these files?

**Comment by ANiknejad** [comment]
hi Amina @aechchiki 
question: what about the google doc here?
https://docs.google.com/document/d/1IuNu3WGTSIhXnJffP_yo7lK2abSgxZQDPJgG1SYF5vI/edit?pli=1

is it updated?

many thanks in advance

**Comment by aechchiki** [comment]
@ANiknejad Yes I think it's the good file... will check it again to see if something has to be updated ;)

**Comment by aechchiki** [comment]
Hi, @ANiknejad @fbastian 

So, I updated the GTEx readme, in both the repo (https://github.com/BgeeDB/expression-annotations/blob/master/annotations/gtex/README_GTEx.md) and the GoogleDocs (https://docs.google.com/document/d/1IuNu3WGTSIhXnJffP_yo7lK2abSgxZQDPJgG1SYF5vI/edit?pli=1)

Tell me what you think about it

**Comment by ANiknejad** [comment]
I started to edit the GoogleDocs, please comment too
@aechchiki 
@fbastian
